app/Filament/Resources/UserResource/Pages/EditUser.php

```diff
@@ -76,7 +76,11 @@ public function form(Form $form): Form
                             ->label('Email address')
                             ->email()
                             ->required()
-                            ->maxLength(255),
+                            ->maxLength(255)
+                            ->unique()
```

Comment:
I think this may cause the edit to fail if you don't change the email address.

You may need to apply `ignoreRecord: true` here

Reply:
@Orrison Done